packages/code-du-travail-frontend/src/outils/common/PrevNextBar.js

```diff
@@ -46,18 +48,24 @@ PrevNextBar.defaultProps = {
 
 export { PrevNextBar };
 
-const { spacings } = theme;
+const { breakpoints, spacings } = theme;
+
+const StyledButton = styled(Button)``;
```

Comment:
unused object

Reply:
nice catch

Reply:
Not sure I but you could do  
```js
${Button} + ${Button} {
  margin-top: ${spacings.base};
}
```

Reply:
I hoped that too but because of the forward ref wrapper it's a no go :/